src/Integration/MefServices/IActiveSolutionTracker.cs

```diff
@@ -28,5 +28,7 @@ internal interface IActiveSolutionTracker
         /// The active solution has changed (either opened or closed).
         /// </summary>
         event EventHandler ActiveSolutionChanged;
```

Comment:
As the class needs to be stateful I have decided not to put any specific event args to the event.